src/main/java/io/quarkus/search/app/indexing/IndexingService.java

```diff
@@ -163,29 +163,35 @@ private void createIndexes() {
 
     private void indexAll() {
         Log.info("Indexing...");
-        try (Rollover rollover = Rollover.start(searchMapping);
-                Closer<IOException> closer = new Closer<>()) {
-            // Reset the database before we start
-            clearDatabaseWithoutIndexes();
+        try (FailureCollector failureCollector = new FailureCollector(indexingConfig.errorReporting().enabled(),
```

Comment:
This should be in `reindex`, as I suggested in my earlier comment, because if creating the indexes fails (for some reason), we'll never even go through this code.